### DISCUSSION TITLE
Redis Config on SSL

### DISCUSSION THREAD

**Discussion by mifrasmm** [redisson/redisson#5636] [General]
HiTeam,

im new to this forum , need you support on configure redis on SSL, currently i have below configuration and its working fine with 6370 port how i can enable SSL using below config 

{
   "singleServerConfig":{
      "idleConnectionTimeout":10000,
      "pingTimeout":1000,
      "connectTimeout":10000,
      "timeout":3000,
      "retryAttempts":3,
      "retryInterval":1500,
      "password":"mypassword",
      "subscriptionsPerConnection":5,
      "clientName":null,
      "address":"redis://mydomain.net6379",
      "subscriptionConnectionMinimumIdleSize":1,
      "subscriptionConnectionPoolSize":50,
      "connectionMinimumIdleSize":32,
      "connectionPoolSize":64,
      "database":0,
      "dnsMonitoringInterval":5000
   },
   "threads":0,
   "nettyThreads":0,
   "codec":{
      "class":"org.redisson.codec.SerializationCodec"
   },
   "transportMode":"NIO"
}

**Comment by mrniko** [comment]
Just use rediss in your url: **rediss**://mydomain.net6379
